Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/ParabolaConstrainedLineDataProvider.cs

```diff
@@ -87,13 +87,19 @@ protected override Vector3 GetPointInternal(int pointIndex)
         /// <inheritdoc />
         protected override void SetPointInternal(int pointIndex, Vector3 point)
         {
-            if (pointIndex == 1)
-            {
-                endPoint.Position = point;
-            }
-            else
+            switch (pointIndex)
             {
-                Debug.LogError("Invalid point index!");
+                case 0:
+                    // We do nothing with this case. Parabola lines start at 0,0,0
+                    // Case 0's point is (0,0,0)
+                    // 10/25/2018 - VNext Refactoring of Tooltips
```

Comment:
This comment isn't needed. Thanks!

Reply:
I'm pretty sure this change already made it into the dev branch. @JonathanPalmerGD please merge the latest changes after merging my PR